// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript5/EnumDeclarations/parserEnumDeclaration6.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -6,4 +6,18 @@
     B = 2,
     C = 2,
     D = 3
 }
+/// [Errors] ////
+
+parserEnumDeclaration6.ts(4,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
+
+
+==== parserEnumDeclaration6.ts (1 errors) ====
+    enum E {
+        A = 1,
+        B,
+        C = 1 << 1,
+        ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
+        D,
+    }
\ No newline at end of file
